tbprofiler/4.3.0/Dockerfile

```diff
@@ -37,12 +37,13 @@ ARG MAMBA_DOCKERFILE_ACTIVATE=1
 # Version of database can be confirmed at ./TBProfiler-${TBPROFILER_VER}/db/tbdb.version.json
 RUN tb-profiler update_tbdb
 
-ARG MAMBA_DOCKERFILE_ACTIVATE=1
+ENV MAMBA_DOCKERFILE_ACTIVATE=1
 WORKDIR /data
 
-FROM app as test
+ENV PATH="/opt/conda/bin/:${PATH}"
```

Comment:
please remove the `/` after `bin` on line 43

Everything still works and this doesn't really matter, but you can see that the addition to the PATH variable is a bit different than the defaults:
```
Singularity> echo $PATH
/opt/conda/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Reply:
Done